### ISSUE TITLE
Adding System with same name should allow retry. 

### ISSUE THREAD

**Issue by rcaloras** [rcaloras/bashhub-client#37]
Got this today when I entered a duplicate name...

``` bash
What do you want to call this system? For example Home, File Server, ect. [some.hostname.com]: An Existing Name

This name is already registered for one of your systems.
Sorry looks like getting your info failed.                    Exiting..
```

The client should prompt for the name again instead of simply bailing out.

**Comment by rcaloras** [comment]
Similar issue to this one. Retry should also be allowed in the client for bad username. Tried to register with punctuation and had to restart.

**Comment by g33k247** [comment]
Also experienced this same issue.

**Comment by g33k247** [comment]
Any movement on a fix here?

**Comment by rcaloras** [comment]
@g33k247 put together a PR to allow for retrying and updated backend error messaging. Feel free to share thoughts:
https://github.com/rcaloras/bashhub-client/pull/92

**Comment by rcaloras** [comment]
Fixed in 2.1.2
